process/accept-orders.py

```diff
@@ -1,23 +1,26 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 import os
 import os.path
-import ConfigParser
+import io
+try:
+  	from configparser import ConfigParser
+except ImportError:
+	from ConfigParser import ConfigParser
 import string
 import logging
 import sys
 import subprocess
 import time
 import socket
-import rfc822
 from stat import ST_MTIME
-from email.Utils import parseaddr
-from email.Parser import Parser
+from email.utils import parseaddr, parsedate_tz, mktime_tz
+try:
```

Comment:
Gibt es einen Grund, dass BytesParser in Python 3 nicht gefunden wird? Wenn ja, wie unterscheiden sich Parser und BytesParser, und welche Probleme treten auf, wenn man wie hier eine Alternative benutzt? Das scheint mir irgendwie unsicher.

Reply:
Auch hier ohne python2 kann das weg.

Reply:
Machst Du das gerade noch?